pkg/proc/proc_test.go

```diff
@@ -1339,6 +1375,9 @@ func TestRecursiveStructure(t *testing.T) {
 }
 
 func TestIssue316(t *testing.T) {
+	if runtime.GOARCH == "arm64" {
+		t.Skip("arm64 do not support Eval Variable for now.")
```

Comment:
I'll be honest. I don't see much of a point in merging an arm64 support patch that can't read from memory.